doc/build/sysbuild/index.rst

```diff
@@ -801,6 +801,27 @@ debug the application.
 .. _MCUboot with Zephyr: https://docs.mcuboot.com/readme-zephyr
 .. _ExternalProject: https://cmake.org/cmake/help/latest/module/ExternalProject.html
 
+.. _sysbuild_board_roots:
+
+Specifying custom board roots with sysbuild
+*******************************************
+
+Because sysbuild is a CMake project in it's own right, specifying ``BOARD_ROOT`` in an
+application ``CMakeLists.txt`` will not work, instead the board root must be set by using a
+:ref:`a module <modules_build_settings>` or by using a custom sysbuild project file, for example
+``<application>/sysbuild/CMakeLists.txt``:
```

Comment:
I understand why you put this first, but I think that when describing a feature in a section, then we start with describing what the feature does / how it works, and not how it doesn't work.

I think this section would be better if used for describing `<application>/sysbuild/CMakeLists.txt` itself, and in that process use `BOARD_ROOT` as an example.

Reply:
Updated